connectors/sources/oracle.py

```diff
@@ -34,16 +39,58 @@ def __init__(self, configuration):
         """
         super().__init__(configuration=configuration)
         self.is_async = False
+        self.oracle_home = self.configuration["oracle_home"]
+        self.wallet_config = self.configuration["wallet_configuration_path"]
+        self.protocol = self.configuration["oracle_protocol"]
         self.dsn = f"(DESCRIPTION=(ADDRESS=(PROTOCOL={self.protocol})(HOST={self.host})(PORT={self.port}))(CONNECT_DATA=(SID={self.database})))"
         self.connection_string = (
-            f"oracle://{self.user}:{quote(self.password)}@{self.dsn}"
+            f"oracle+oracledb://{self.user}:{quote(self.password)}@{self.dsn}"
         )
         self.queries = QUERIES
         self.dialect = "Oracle"
 
+    @classmethod
+    def get_default_configuration(cls):
+        """Get the default configuration for database-server configured by user
+
+        Returns:
+            dictionary: Default configuration
+        """
+        oracle_configuration = copy.deepcopy(super().get_default_configuration())
```

Comment:
nit: since we don't change the items, you can probably just use `super().get_default_configuration().copy()` (shallow)

Reply:
done